addons/account_asset/account_asset.py

```diff
@@ -61,8 +64,9 @@ class account_asset_category(osv.osv):
         'method': 'linear',
         'method_number': 5,
         'method_time': 'number',
-        'method_period': 12,
         'method_progress_factor': 0.3,
+        'type': 'sales',
```

Comment:
The default type must be "Purchase", not "sales". I also suggest to rename "Sales" into "sale"